libs/langchain/langchain/schema/runnable/fallbacks.py

```diff
@@ -0,0 +1,286 @@
+import asyncio
+from typing import (
+    TYPE_CHECKING,
+    Any,
+    Iterator,
+    List,
+    Optional,
+    Sequence,
+    Tuple,
+    Type,
+    Union,
+)
+
+from langchain.load.dump import dumpd
+from langchain.pydantic_v1 import BaseModel
+from langchain.schema.runnable.base import Runnable, RunnableSerializable
```

Comment:
`is_lcserializable` is a class method, so I think inheritance will work, we could define it such that anything that inherits from RunnableSerializable is in fact serializable and also provide a default namespace